File: C/Pattern_6.c
#include<stdio.h>
void main()
{
	int n,i,j,k;
	printf("Enter height : ");
	scanf("%d",&n);
	printf("\n");
	for(i=1;i<=n;i++)
	{
		for(k=i;k<n;k++)
		{
			printf(" ");
		}
		for(j=1;j<=i;j++)
		{
			printf(" %d",j);
		}
		printf("\n");
	}
}

File: Java/BlueJ/Java_files/Pattern6.java
import java.io.*;
public class Pattern6
{
    public static void main(String args[])throws IOException
    {
        BufferedReader br=new BufferedReader(new InputStreamReader(System.in));
        int h,i,j;
        System.out.print("Enter height:");
        h=Integer.parseInt(br.readLine());
        for(i=1;i<=h;i++)
        {
            for(j=i;j<=h;j++)
            {
                System.out.print(j+" ");
            }
            System.out.println();
        }
    }
}
